src/modules/ChannelList/dux/reducers.ts

```diff
@@ -276,24 +276,6 @@ export default function channelListReducer(
           }),
         };
       })
-      .with({ type: channelListActions.CHANNEL_REPLACED_TO_TOP }, (action) => {
```

Comment:
why is this removed?

Reply:
The channel item will be sorted by the `channelListQuery.order`, so we don't have to move the channel when sending a message.

same as below